src/Core/Enums/PlanType.cs

```diff
@@ -28,14 +28,24 @@ public enum PlanType : byte
     EnterpriseMonthly2020 = 10,
     [Display(Name = "Enterprise (Annually) 2020")]
     EnterpriseAnnually2020 = 11,
+    [Display(Name = "Teams (Monthly) 2023")]
+    TeamsMonthly2023 = 12,
+    [Display(Name = "Teams (Annually) 2023")]
+    TeamsAnnually2023 = 13,
+    [Display(Name = "Enterprise (Monthly) 2023")]
+    EnterpriseMonthly2023 = 14,
+    [Display(Name = "Enterprise (Annually) 2023")]
+    EnterpriseAnnually2023 = 15,
+    [Display(Name = "Teams Starter 2023")]
+    TeamsStarter2023 = 16,
     [Display(Name = "Teams (Monthly)")]
-    TeamsMonthly = 12,
+    TeamsMonthly = 17,
```

Comment:
💭 I thought this wrong for a second, but this is how we grandfather in existing organizations, right? They're DB column will still point to the 2023 ints; clever.

Reply:
Yeah, exactly. This will only affect new organizations